# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sash/hotfix-no-strip.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sash-3.8/Makefile.vanilla	2022-03-31 15:48:14.367396414 +0200
+++ sash-3.8/Makefile	2022-03-31 15:48:18.954396650 +0200
@@ -54,7 +54,7 @@
 
 sash:	$(OBJS)
 	$(CC) $(LDFLAGS) -o sash $(OBJS) $(LIBS)
-	strip sash
+#	strip sash
 
 clean:
 	rm -f $(OBJS) sash
